suricata.yaml.in

```diff
@@ -660,6 +660,68 @@ af-packet:
     #use-mmap: no
     #tpacket-v3: yes
 
+dpdk:
+  eal-params:
+    proc-type: primary
+    l: 0-6
```

Comment:
Suggest comments here and use of long-form options (if applicable)

Reply:
just `l` is not a great variable name. Depending on font its hard to distinguish from `1`

Reply:
as mentioned in v3 and v4 rte_remote_launch is not mandatory for running DPDK l;library please rework to remove such configuration as it conflict with thread management of suricata

Reply:
removed